chipsec/modules/common/uefi/access_uefispec.py

```diff
@@ -257,7 +257,8 @@ def run(self, module_argv: List[str]) -> int:
         attribute = None
 
         if len(module_argv) > 1:
-            attribute = module_argv[1]
+            _attr = module_argv[1]
+            attribute = int(_attr, base=16) if _attr[:2] == "0x" else int(_attr)
```

Comment:
Please use `int(_attr, base=0)` As this will do the same thing. (Thanks for catching this one!)